src/domain/services/AIService.ts

```diff
@@ -358,29 +359,20 @@ export class AIService {
         whereCondition.AND.push(this.getGroupCriteria(orgId, groupId));
       }
     }
-    if (categoryId) {
-      whereCondition.AND.push(this.getCategoryCriteria(categoryId));
-    }
     if (search) {
       whereCondition.AND.push(this.getSearchCriteria(search));
     }
-
+    if (categoryId) {
+        whereCondition.AND.push(this.getCategoryCriteria(categoryId));
+    }
     if (approvedByOrg !== null && approvedByOrg !== undefined) {
       whereCondition.AND.push(
         this.getApprovedByOrgCriteria(orgId, approvedByOrg)
       );
     }
 
     const ais = await prismadb.aI.findMany({
-      select: {
-        ...listAIResponseSelect(),
-        chats: {
-          where: {
-            userId,
-            isDeleted: false,
-          },
-        },
-      },
```

Comment:
I hadn't realized earlier that chats were already included in the select. Let's keep it as it was.

Reply:
Added